src/modules/customers/dto/dtfs-customer.dto.ts

```diff
@@ -13,4 +13,9 @@ export class DTFSCustomerDto {
   @IsString()
   @IsNotEmpty()
   companyName: string;
+
+  @ApiProperty({ description: 'Probability of Default' })
+  @IsNumber()
+  @IsNotEmpty()
```

Comment:
Will this accept `14.1` as a default PoD from DTFS?

Reply:
Will add a test for this